apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -61,10 +61,10 @@ export const UnconnectedTeacherHomepage = ({
   const teacherReminders = useRef(null);
   const flashes = useRef(null);
 
-  /* We are hiding the AFE banner to free up space on the Teacher Homepage as of September 2023).
-   * When we want to show the AFE banner again remove the next line and uses of 'shouldShowAFEBanner'.
+  /*
+   * Determines whether the AFE banner will take premium space on the Teacher Homepage
    */
-  const shouldShowAFEBanner = false;
+  const shouldShowAFEBanner = true;
```

Comment:
given your work to translate the banner, should [this line](https://github.com/code-dot-org/code-dot-org/blob/71ccd835bf1f182cb07ee155177b96ca08014850/apps/src/templates/studioHomepages/TeacherHomepage.jsx#L164) be updated to no longer restrict it to english?

Reply:
(if you do update that, can you also update the name of the test below?)

Reply:
Oo good question... @dmcavoy is this the kind of change that should happen now or once we've gotten translations?

Reply:
I think we should just update it now and then translations will follow. Better to give teachers the ability to apply than restrict it.

Reply:
Roger that! I made the updates. I also removed that prop from TeacherHomepage entirely (and the corresponding calls, story and test files) because nothing else was using it.